MonoGame.Framework/Platform/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -187,6 +187,30 @@ protected virtual void CreateDeviceResources()
                 creationFlags |= SharpDX.Direct3D11.DeviceCreationFlags.Debug;
             }
 
+#if WINDOWS_UAP
+            // Check if tearing is supported
+            if (PresentationParameters.PresentationInterval == PresentInterval.Immediate)
+            {
+                RawBool allowTearing;
+                using (var dxgiFactory2 = new Factory2())
+                {
+                    unsafe
+                    {
+                        var factory5 = dxgiFactory2.QueryInterface<Factory5>();
```

Comment:
This line should go inside the `try ... catch` block in case the interface isn't supported.

Reply:
The reason why it's not within a try catch is because that factory interface was introduced on the Anniversary SDK (14393). MonoGame doesn't support a lower version, so I just decided not to do a try catch because it shouldn't ever throw an exception. Do you want me to do it nevertheless?